docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -47,11 +47,10 @@ settings, see {ref}/ml-settings.html[{ml-cap} settings in {es}].
 The runtime of {dfanalytics-jobs} depends on numerous factors, such as the
 number of data points in the data set, the type of analytics, the number of 
 fields that are included in the analysis, the supplied 
-{ref}/put-dfanalytics.html#ml-hyperparam-optimization[hyperparameters], the 
-type of analyzed fields, and so on. For this reason, a general runtime value that 
-applies to all or most of the situations does not exist. The runtime of a 
-{dfanalytics-job} may take from a couple of minutes up to 35 hours in extreme 
-cases.
+<<hyperparameters,hyperparameters>>, the type of analyzed fields,
+and so on. For this reason, a general runtime value that applies to all or most
+of the situations does not exist. The runtime of a {dfanalytics-job} may take
+from a couple of minutes up to 35 hours in extreme cases.
```

Comment:
I wouldn't be as specific here (it implies there is a hard limit which there isn't):
```suggestion
from a couple of minutes up to many hours in extreme cases.
```